src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -29,15 +29,16 @@ class InputOption
     private $name;
     private $shortcut;
     private $mode;
+    /** @var string|string[]|bool|null */
```

Comment:
We usually don't add doc blocks for types that can be inferred from the constructor, to reduce the visual noise.

Reply:
Done.